dasharo-compatibility/device-detection.robot

```diff
@@ -0,0 +1,105 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+# TODO: maybe have a single file to include if we need to include the same
+# stuff in all test cases
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+Resource            ../keys.robot
+Resource            ../pikvm-rest-api/pikvm_comm.robot
+Resource    ../lib/options/dcu.robot
+
+# TODO:
+# - document which setup/teardown keywords to use and what are they doing
+# - go threough them and make sure they are doing what the name suggest (not
+# exactly the case right now)
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+...                     AND
+...                     Set UEFI Option    UsbDriverStack    Enabled
+Suite Teardown      Run Keywords
+...                     Log Out And Close Connection
+...                     AND
+...                     Set UEFI Option    UsbDriverStack    Enabled
+
+
+*** Test Cases ***
+# DDET001.001 USB Stack disable
+#     [Documentation]    Test disabling the USB stack
+
+#     Login To Linux With Root Privileges
+#     Set UEFI Option    UsbDriverStack    Disabled
+
+#     Execute Reboot Command
+#     Sleep    10s
+
+#     Set DUT Response Timeout    30s
+#     Login To Linux With Root Privileges
+#     ${out}=    Execute Command In Terminal
+#     ...    cbmem -1 | grep "UsbBusStart:"
+
+#     Should Contain    ${out}     usb bus started
+#     ...    ignore_case=True
+
+# DDET002.001 USB Stack enable
```

Comment:
Should these remain commented out?

Reply:
Yes, they won't work because of the issues with a python library. Trying to write anything via ssh after disabling usb stack will cause an exception and end the whole test suite.

Reply:
Right, @macpijan any ideas what might be going wrong in the paramiko library?